holidayextras/brands
# Asset deployment

#### What does this PR do? (please provide any background)
Enables us to deploy assets, e.g. fonts and images alongside the brand styling. Added notes to the readme for further explanation but essentially when you make a change to the brand, it will deploy the recently added files to the S3 bucket as soon as your branch is merged to master.

#### What tests does this PR have?
None

#### How can this be tested?
Grab the branch, add an asset as per readme and then run gulp in the root of the project. You should see a `dist` directory in the root with your newly added assets.

#### Screenshots / Screencast
<img width="468" alt="screen shot 2016-03-31 at 12 42 58" src="https://cloud.githubusercontent.com/assets/6357978/14174715/2266ab18-f73e-11e5-8b64-5c614a815951.png">


#### What gif best describes how you feel about this work?
![](https://media2.giphy.com/media/detDESm0bahoc/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
